doc/en/user/source/datadirectory/location.rst

```diff
@@ -1,29 +1,96 @@
 .. _datadir_location:
 
-Data directory default location
-===============================
+Data directory location
+=======================
+
+The current data directory location is always shown on the :ref:`config_serverstatus` page.
+
+.. figure:: /configuration/img/server_status.png
+   
+   Status Page (default tab)
+   
+Default data directory location
+-------------------------------
+
+By default GeoServer includes an example data directory allowing you to try out the application quickly:
+
+* Platform Independent Binary: The data directory is located at :file:`<installation root>/data_dir`.
+
+  .. list-table::
+     :header-rows: 1
+  
+     * - Platform
+       - Default location
+     * - Linux 
+       - :file:`/var/lib/tomcat9/webapps/geoserver/data_dir`
+     * - Windows
+       - :file:`C:\\Program Files\\Apache Software Foundation\\Tomcat 9.0\\webapps\\geoserver\\data`
```

Comment:
Does the "Platform Independent Binary" have **any** default directory for the data_dir, because it is up to the user where they decide to unzip GeoServer? It is just :file:`<installation root>/data_dir`.

Reply:
The script checks for GEOSERVER_DATA_DIR  and supplies the value of `<installation root>/data_dir` if it was otherwise unset.